lambdatimes/src/components/Content/Content.js

```diff
@@ -48,7 +54,11 @@ export default class Content extends Component {
           `selectedTab` that includes the currently selected tab
           and `selectTabHandler` that includes the function to change the selected tab
         */}
-        <Tabs tabs={this.state.tabs} />
+        <Tabs 
+          tabs = {this.state.tabs} 
+          selectedTab = {this.state.selected}
+          selectTabHandler = {this.state.changeSelected} />
```

Comment:
SOLVED: MVP complete.
```suggestion
          selectTabHandler = {this.changeSelected} />
```